website/content/docs/upgrading/upgrade-to-1.19.x.mdx

```diff
@@ -42,6 +42,23 @@ based on the table below.
 | CE            | true      | any value other than sha2-512 | An error is returned                       | Pure Ed25519                      |
 | CE            | true      | sha2-512                      | An error is returned (not supported on CE) | Pure Ed25519                      |
 
+### Identity System Duplicate Cleanup
+
+**Users should review their server logs after upgrading to see if identity
```

Comment:
You explained the 1.19 changes/recommendations in this section in a simple and concise way, great job! 😀 

Only recommendation I have is that I don't think we need this bold part because we already essentially say the same thing down below in the "also includes improved reporting" paragraph, or if you want to keep it, at least just moving it to inside or after that paragraph, so all mention of the server logs is in the same place. 

Otherwise the bolded line coming first kind of looks like we're saying we've added identity duplicates to 1.19 as some kind of regression.

Reply:
Y'know, I wrote this comment at the beginning of my review, and now I'm seeing it again after having read the rest of the doc, and I'm not sure I agree with myself. Choose what makes the most sense to you!